homeassistant/components/sensor/waqi.py

```diff
@@ -77,7 +77,7 @@ def name(self):
         try:
             return self._details['city']['name']
```

Comment:
This one returns the actual name of the station, for example, 'Beijing US embassy'. Actually currently, adding a prefix makes every station in UI starts with WAQI, which looks a bit redundant from my view:). Would it makes more sense to keep it as is provided?  Otherwise, when a station name is added to yaml, we need either subtract prefix from self.name, or add prefix to CONF_station, both way seems unnecessary.